tools/amp_segment/ina_speech_carbonate.xml

```diff
@@ -0,0 +1,24 @@
+<tool id="inaspeech_carbonate" name="INA Speech Segmenter on Carbonate" version="1.0.2">
+  <description>INA Speech Segmenter on IU's HPC</description>
```

Comment:
the non HPC version has id ina_speech_segmenter. how about we call the HPC one either ina_speech_segmenter_hpc, or simply ina_hpc.